curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5632.md

```diff
@@ -0,0 +1,79 @@
+---
+id: 66fbcf750a62784cf11f5632
+title: Step 8
+challengeType: 1
+dashedName: step-8
+---
+
+# --description--
+
+Create a `getTotalIngredients` function with an `ingredients` parameter. This function should return the length of the total ingredients of each recipe, which is a number.
+
+# --hints--
+
+You should create a `getTotalIngredients` function.
+
+```js
+assert.isFunction(getTotalIngredients)
+```
+
+You `getTotalIngredients` function should have an `ingredients` parameter.
+
+```js
+assert.match(getTotalIngredients.toString(), /ingredients/);
+```
+
+Your `getTotalIngredients` function should return a number.
```

Comment:
same issue here

You need to have additional tests to prevent campers from doing this

<img width="762" alt="Screenshot 2024-10-10 at 5 16 16 PM" src="https://github.com/user-attachments/assets/a297497f-8c1a-4b05-910b-ea585da9e068">

Reply:
Can you check again to see if what I have is good enough?

Reply:
We should have a general hint like ```Your `getTotalIngredients` function should return the correct number of ingredients.``` Or something like that.

Then you can have several tests with multiple functions like we are doing in other workshops like here:

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/workshop-calculator/66cc14ca14e65e7a9ebecb80.md

Reply:
aside from that change, this workshops will be good to go 👍

Reply:
Should be fine now